core/src/main/java/org/hibernate/ogm/type/impl/TypeTranslatorImpl.java

```diff
@@ -35,11 +35,14 @@
 import org.hibernate.type.descriptor.java.JdbcTimestampTypeDescriptor;
 import org.hibernate.type.descriptor.java.LongTypeDescriptor;
 import org.hibernate.type.descriptor.java.PrimitiveByteArrayTypeDescriptor;
+import org.hibernate.type.descriptor.java.ShortTypeDescriptor;
 import org.hibernate.type.descriptor.java.StringTypeDescriptor;
 import org.hibernate.type.descriptor.java.UUIDTypeDescriptor;
 import org.hibernate.type.descriptor.java.UrlTypeDescriptor;
 import org.hibernate.usertype.UserType;
 
+import static org.hibernate.ogm.util.impl.CollectionHelper.newHashMap;
+
```

Comment:
We try to avoid this kind of changes in a pull request since they are unrelated to the issue.
We also tend to keep the staic import before the regular imports